mountpoint-s3/tests/fuse_tests/write_test.rs

```diff
@@ -672,7 +672,7 @@ fn multi_thread_test() {
     .unwrap();
 }
 
-fn overwrite_test<F>(creator_fn: F, prefix: &str)
+fn overwrite_test<F>(creator_fn: F, prefix: &str, write_only: bool)
```

Comment:
What's the purpose of this test? Overwrite with truncate, no fsync?

Reply:
This is a basic test that overwrite works. Truncate is required to overwrite a file and fsync is used in the test so that the subsequent read is guaranteed to be consistent.

Reply:
Ah, got it. Happy path.